src/docs/asciidoc/reference_doc/card_structure.adoc

```diff
@@ -210,22 +220,5 @@ card:
 In this sample, the card will be displayed twice in the time line. The card
 start date will be ignored.
 
-*example 2:*
-
-Instead of the default clustered view, you may want your card to be displayed
-as a line in the time line.
-
-....
-{
-	"publisher":"TSO1",
-	"publisherVersion":"0.1",
-	"processId":"process-000",
-	"startDate":1546297200000,
-	"severity":"INFORMATION",
-	...
-	"timeSpans" : [
-        {"start" : 1546297200000, "end" : 1546297500000}
-    ]
 
-}
-....
+For timeSpans, you can specify an end date but it is not used by operator fabric (it was intended for future uses but it will be deprecated)
```

Comment:
```suggestion
For timeSpans, you can specify an end date but it is not implemented in OperatorFabric (it was intended for future uses but it will be deprecated).
```